.github/workflows/test.yml

```diff
@@ -80,7 +90,7 @@ jobs:
         # image layer.
         install: |
           case "${{ matrix.distro }}" in
-            ubuntu*|jessie|stretch|buster|bullseye)
+            ubuntu*|jessie|stretch|buster|bullseye|unstable|experimantal)
```

Comment:
Is the typo (experimantal instead of experimental) intentional here?

Reply:
Thanks, fixed.